wiki/quest-modding.md

```diff
@@ -10,11 +10,11 @@ For new users, BeatSaberQuestInstaller may be easier to use as it handles some o
 
 ### Installing with BeatSaberQuestInstaller
 
-Before starting the process go to the utilities tab and backup your scores, if you've previously modded Beat Saber you also need to go there to uninstall the game, make sure you backup **BEFORE** uninstalling.
+**Make sure you run the installer as administrator**, if you're a new user follow all the steps until you've done step 3, after that go to the utilities tab and backup your scores, if you've previously modded Beat Saber you also need to go there to uninstall the game, make sure you backup **BEFORE** uninstalling.
```

Comment:
I think this would also fit as a warning block

Such as:
::: warning 
Make sure you run the installer as administrator
:::